subxt/src/rpc/rpc.rs

```diff
@@ -446,6 +450,163 @@ impl<T: Config> Rpc<T> {
             self.client.request("system_dryRun", params).await?;
         Ok(types::decode_dry_run_result(&mut &*result_bytes.0)?)
     }
+
+    /// Subscribe to `chainHead_unstable_follow` to obtain all reported blocks by the chain.
+    ///
+    /// The subscription ID can be used to make queries for the
+    /// block's body [`chainhead_unstable_body`](Rpc::chainhead_unstable_follow),
```

Comment:
```suggestion
    /// block's body ([`chainhead_unstable_body`](Rpc::chainhead_unstable_follow)),
```

For consistency with the others :)